### PR TITLE
Align icons & text in SideNav and Dropdown

### LINKED ISSUE #1484: SideNav: Icons and text are misaligned
## Bug Report

**Current Behavior**
The icons and the text ("All Discussions", "Categories" etc.) are not aligned well (see screenshots).

**Steps to Reproduce**
1. Go to the homepage
2. See bug.

**Expected Behavior**
Text should be aligned right to the center of the icons.

**Screenshots**

 

**Environment**
Flarum Community
https://discuss.flarum.org

### PR DESCRIPTION
**Pull request by datitisev**
**Fixes #1484**

**Changes proposed in this pull request:**
- Removes margin top & makes line height 1 in sideNav icon & Dropdown icon

**Confirmed**

- [x] Frontend changes: tested on a local Flarum installation.
- [ ] Backend changes: tests are green (run `php vendor/bin/phpunit`).

### CODE REVIEW

**Comment by ardacebi** [comment]
I already opened #1542.

**Comment by ardacebi** [comment]
FA 5 seems to fix this issue for icons. I mean, it isn't fixed completely and this PR is still necessary but just wanted to share.

Old:

screenshot_2018-08-13 flarum community

Latest dev-master:

screenshot_2018-08-13 locflar

You can see the difference (for the icons).

File: less/common/sideNav.less [hunk 1]
```diff
@@ -30,8 +30,8 @@
           & .Button-icon {
             float: left;
             margin-left: -30px;
-            margin-top: 1px;
             font-size: 15px;
+            line-height: initial;
```
Comment (tobscure) [inline less/common/sideNav.less]: Both of the `line-height: initial` rules are not actually doing anything for me. Can we remove them?

Comment (datitisev) [reply less/common/sideNav.less]: Hm, sure. I remember seeing a slight change with the `line-height` rules, but I can no longer reproduce them.

**Pull request merged**